README.md

```diff
@@ -45,16 +44,14 @@ architecture listed below.
 
 * CentOS 7
 * Debian 7
-* (Debian 8)
-* (Debian 9)
 * FreeBSD 10.3
 * FreeBSD 11.1
 * Ubuntu 14.04
 * Ubuntu 16.04
 
 _Currently, Debian 8 and Debian 9 are not supported due to a known bug in Zonemaster::LDNS, see [Zonemaster-LDNS issue 10] and [Zonemaster-LDNS issue 30]. 
-That has to be resolved 
-before we can include support for Debian 8 and Debian, respectively._
+These have to be resolved 
```

Comment:
"That" refer to the know bug (one) rather than the OSs (two). On the other hand, the bug seems to be resolved so this section should probably be removed when we update.

Reply:
Should we really keep two issues open for a single bug? Wouldn't it be better to close one of them as a duplicate of the other?